test/CMakeLists.txt

```diff
@@ -110,6 +111,20 @@ if(SINGULARITY_BUILD_CLOSURE)
                                          singularity-eos::singularity-eos)
     target_include_directories(test_pte PRIVATE ${CMAKE_CURRENT_SOURCE_DIR})
     add_test(pte test_pte)
+##### 
```

Comment:
Extraneous comment?

Reply:
Oops, how did that get into the file? Took it away.